app/internal/page/fixtures.go

```diff
@@ -72,6 +72,10 @@ func Fixtures(tmpl template.Template) Handler {
 						values.Add("startCpFlowDonorHasNotPaid", "1")
 					}
 
+					if data.Form.Signed != "" {
+						values.Add("lpa.signedByDonor", "1")
```

Comment:
are we saying that the LPA will always be signed for the flow to start now, so we could always set this?

Reply:
Not necessarily - from what I understand it will still be possible for a CP to start the flow while the donor has not signed. In this case the task list will have two sections rather than one and they'll be unable to get to the flow where they see the what happens next page and signing page. They'll have an option to view the draft LPA but all other section links will show as cannot start)